templates/CRM/Contribute/Form/AdditionalPayment.tpl

```diff
@@ -100,7 +100,7 @@
             </td>
           </tr>
           <tr class="crm-payment-form-block-payment_instrument_id">
-            <td class="label">{$form.payment_instrument_id.label}</td>
+            <td class="label">{$form.payment_instrument_id.label} <span class="crm-marker" title="{ts}This field is required.{/ts}">*</span></td>
```

Comment:
@pradpnayak why can't we change at backend by setting required parameter TRUE for  ```payment_instrument_id``` [here](https://github.com/civicrm/civicrm-core/blob/master/CRM/Contribute/Form/AdditionalPayment.php#L255) ?